pages.ko/common/rm.md

```diff
@@ -0,0 +1,24 @@
+# rm
+
+> 파일 혹은 폴더를 삭제.
+> 더 많은 정보: <https://www.gnu.org/software/coreutils/rm>.
+
+- 임의의 경로에서 파일을 제거:
+
+`rm {{path/to/file}} {{path/to/another/file}}`
```

Comment:
Thanks for contribution. Please translate `path/to/something` into Korean as well.

Reply:
resolved!